setup.py

```diff
@@ -92,6 +99,8 @@ def build_extension(self, ext):
                 env.get('CXXFLAGS', ''), self.distribution.get_version())
             if not os.path.exists(self.build_temp):
                 os.makedirs(self.build_temp)
+            os.environ["CXXFLAGS"] = "-fPIC"
```

Comment:
This looks good, maybe add a comment about why this is here?

Reply:
Worth doing the equivalent of `CXXFLAGS += -fPIC`?

Reply:
Good idea.

Reply:
Added a comment, and included previous flags from the environment.